packages/editor/src/components/toolbar/Toolbar2.tsx

```diff
@@ -83,7 +83,7 @@ const generateToolbarMenu = () => {
     },
     {
       name: t('editor:menubar.saveScene'),
-      hotkey: `${cmdOrCtrlString}+s`,
+      // hotkey: `${cmdOrCtrlString}+s`,
```

Comment:
why did we comment this out?
have not seen the design yet but if the hotkey is absent in designs, let's remove the `hotkey` key value